### DISCUSSION TITLE
文档问题

### DISCUSSION THREAD

**Discussion by OldSmokeGun** [beyondstorage/go-storage#540] [Q&A] [answered]
缺少使用文档，不知道如何安装，如何使用

**Comment by Xuanwo** [comment, marked as answer]
只需要把这个库添加为项目依赖即可：

```
go get github.com/aos-dev/go-storage/v3
```

quick start 可以参见 https://github.com/aos-dev/go-storage#quick-start

我们现在正在写一些 example，可以关注一下我们的文档官网： https://aos.dev/docs/go-storage/index

**Reply by OldSmokeGun** [reply]
如何使用腾讯的 cos 啊？还有腾讯的 cos 只支持 v1？

**Reply by Xuanwo** [reply]
Hi, 我这边创建了一个 example 项目，可以参考一下：https://github.com/aos-dev/go-storage-example

有看什么样的示例也可以提出来，我们一并加到 example 项目里面

**Reply by Xuanwo** [reply]
如果还是有问题的话可以来我们的中文 Matrix 群组： https://matrix.to/#/#go-storage-zh:aos.dev 实时沟通

**Comment by OldSmokeGun** [comment]
go-storage 到了 v3，下载腾讯 cos 的 service 报错啊

**Reply by Xuanwo** [reply]
cos 的支持目前还没 release, 需要这样来 get：

```go
go get github.com/aos-dev/go-storage/v3
go get github.com/aos-dev/go-service-cos@master
```
